src/server/scripts/Kalimdor/Firelands/instance_firelands.cpp

```diff
@@ -18,6 +18,18 @@
 #include "ScriptMgr.h"
 #include "InstanceScript.h"
 #include "firelands.h"
+#include <minwindef.h>
```

Comment:
OK. For a moment I thought it was a `mingw` file because of all the `mingw` hits in Google search, but it is actually `C:\Program Files (x86)\Windows Kits\8.1\Include\shared\minwindef.h` and therefore not available unless you compile on Windows.